### PR TITLE
User story 31

### PR DESCRIPTION
**Pull request by ggerdsen**
Story is working, test could use some tests for edge cases with multiple merchants.

User Story 31, All Merchants fulfill items on an order

When all items in an order have been "fulfilled" by their merchants
The order status changes from "pending" to "packaged"

### CODE REVIEW

File: app/views/merchant/orders/show.html.erb [hunk 1]
```diff
@@ -39,7 +39,7 @@
         <td><p><%= number_to_currency(item_order.price)%></p></td>
         <td><p><%= item_order.quantity%></p></td>
         <td><p><%= number_to_currency(item_order.subtotal)%></p></td>
-        <td><%= button_to "Fulfill Item", "/merchant/orders/#{item_order.id}", method: :patch %></td>
+        <td><%= button_to "Fulfill Item", "/merchant/orders/#{@order.id}/items/#{item_order.item.id}/update", method: :patch %></td>
```
Comment (JBelta) [inline app/views/merchant/orders/show.html.erb]: Had the same syntax, Just trouble matching css class in the spec "within "#class"

File: spec/features/merchant/orders/update_spec.rb [hunk 2]
```diff
@@ -47,19 +41,23 @@
...
-
+    expect(@order.status).to eq("pending")
+    
     click_on("Fulfill Item", match: :first)
-    binding.pry
-
-
-    #expect(page).to have_content("packaged")
+    
+    @order = Order.first
```
Comment (JBelta) [inline spec/features/merchant/orders/update_spec.rb:58]: Avoids ambiguous error?
